examples/rgbpp/xudt/btc-transfer-all/1-btc-transfer-all.ts

```diff
@@ -0,0 +1,75 @@
+import { bitcoin } from 'rgbpp/btc';
+import { buildRgbppTransferAllTxs, sendRgbppTxGroups } from 'rgbpp';
+import { btcDataSource, isMainnet, collector, btcAccount } from '../../env';
+import { signPsbt } from '../../shared/btc-account';
+import { saveCkbVirtualTxResult } from '../../shared/utils';
+
+interface TestParams {
+  xudtTypeArgs: string;
+  fromAddress: string;
+  toAddress: string;
+}
+const SEND_TX_GROUPS = false;
+
+const rgbppTransferAllTxs = async ({ xudtTypeArgs, fromAddress, toAddress }: TestParams) => {
+  const result = await buildRgbppTransferAllTxs({
+    ckb: {
+      xudtTypeArgs,
+      collector,
+    },
+    btc: {
+      assetAddresses: [fromAddress],
+      fromAddress: fromAddress,
+      toAddress: toAddress,
+      dataSource: btcDataSource,
+      feeRate: 5,
+    },
+    isMainnet,
+  });
+
+  console.log('result.transactions.length', result.transactions.length);
+  console.log('result.summary.included.assets', result.summary.included.xudtAssets);
+  console.log('result.summary.excluded.assets', result.summary.excluded.xudtAssets);
+
+  const signedGroups = await Promise.all(
+    result.transactions.map(async (group) => {
+      const psbt = bitcoin.Psbt.fromHex(group.btc.psbtHex);
+
+      // Sign transactions
+      await signPsbt(psbt, btcAccount);
+
+      psbt.finalizeAllInputs();
+      psbt.extractTransaction();
+
+      return {
+        ckbVirtualTxResult: JSON.stringify(group.ckb.virtualTxResult),
+        btcTxHex: psbt.extractTransaction().toHex(),
+      };
+    }),
+  );
+
+  const ckbVirtualTxResult = JSON.parse(JSON.stringify(signedGroups, null, 2));
```

Comment:
What do you want to save to the local file? `ckbVirtualTxResult` or `signedGroups`?

Seems like the variable naming here is a bit confusing.